README.md

```diff
@@ -1,2 +1,87 @@
-# sb1-open-api-tutorial
-This application shows how one sets up a connection to the SpareBank1 API, getting an oauth token and retrieving  data from a sandbox or production environment.
+# SpareBank1 Open API Tutorial
+
+## Introduction
+This application shows how one sets up a connection to the SpareBank1 API, getting an oauth token and retrieving 
+data from a sandbox or production environment.
+ 
+This example checks for the latest salary that has been put into an account and transfers the remaining balance to 
+another account from the same user. First, its sends a request to find all available accounts. Then you can configure 
+which accounts are going to be used for the transfer of remaining balance.
+
+## Build Application
+```` 
+mvn clean install
+````
+## Set up for sandbox environment  
+### 1. Register application
+Go to the sandbox environment [developersparebank1](https://developersparebank1.no/). Create a user and register your 
+application
+
+### 2. Configure properties for sandbox
+Properties needed to run this application can be found [application.yaml](src/main/resources/application.yaml)
+
+    url:  https://developer-api.sparebank1.no
+    client_id: found where you registered you application
+    client_secret: found where you registered you application
+    redirect_url: the same redirect url that you registered with your application
+    authorization_code: Only used in production. Leave it blank for sandbox
+    from_account_id: running the application will print all accounts. Use the 'id' field from the response
+    to_account_id: running the application will print all accounts. Use the 'id' field from the response
+    
+## Set up for production environment
+### 1. Register Application
+Go to [Personlig Klient](https://developersparebank1.no/personlig-klient) and choose the bank that you are a customer of
+and register the application there. If you're not a customer in any of the SpareBank1 banks, then create a customer 
+relation here https://www.sparebank1.no. 
+
+### 2. Get an authorization code
+Only needed to get data for the production environment. Open in a browser: 
+
+    https://api.sparebank1.no/oauth/authorize
+    ?finInst=fid-smn
+    &client_id=0f603d09-636f-4b3e-96fd-d56dc7d1a1a3
+    &state=3138229
+    &redirect_uri=https%3A%2F%2Fthisisyou.com
+    &response_type=code
+
+
+where
+
+    finInst: is your bank's identifier
```

Comment:
Noe om at 
List of finInst can be found here: https://developersparebank1.no/personlig-klient